src/components/ProjectList/listOfProjects.js

```diff
@@ -196,14 +196,19 @@ const projectList = [
     projectLink: 'https://github.com/danthareja/contribute-to-open-source/contribute',
     description: 'Learn GitHub\'s pull request process by contributing code in a fun simulation project.',
     tags: ['GitHub', 'Tutorial'],
-  }, {
+  },{
     name: 'Mail For Good',
     imageSrc: 'https://avatars0.githubusercontent.com/u/9892522?v=3&s=100',
     projectLink: 'https://github.com/freeCodeCamp/mail-for-good/contribute',
     description: 'An open source email campaign management tool.',
     tags: ['Nodejs', 'JavaScript', 'Email-Campaigns'],
-  },
-  {
+  },{
+    name: 'Hyper',
+    imageSrc: '68747470733a2f2f6173736574732e76657263656c2e636f6d2f696d6167652f75706c6f61642f76313534393732333834362f7265706f7369746f726965732f68797065722f68797065722d332d7265706f2d62616e6e65722e706e67',
```

Comment:
Broken image link, please fix.

Reply:
Hello @sara-02  gentle reminder, I've updated the changes, can you please review and merge the pr.
Thanks in advance